package.json

```diff
@@ -35,18 +35,20 @@
   },
   "homepage": "https://github.com/mcmath/gulp-transform#readme",
   "dependencies": {
-    "@types/node": "*",
-    "@types/gulp-util": "*",
-    "gulp-util": "^3.0.8",
-    "tslib": "^1.7.1"
+    "plugin-error": "^1.0.1",
+    "tslib": "^1.7.1",
+    "vinyl": "^2.1.0"
   },
   "devDependencies": {
     "@types/chai": "^4.0.1",
     "@types/chai-as-promised": "0.0.31",
     "@types/lodash": "^4.14.71",
     "@types/mocha": "^2.2.41",
+    "@types/node": "*",
+    "@types/plugin-error": "^0.1.0",
```

Comment:
`plugin-error` has its own types: no need for the old `@types/plugin-error` package.

Reply:
thanks @demurgos, removed